tf2onnx/onnx_opset/math.py

```diff
@@ -697,3 +697,38 @@ def atan2(y, x):
             op_name_scope=node.name + 'all',
             shapes=[shape], dtypes=[onnx_dtype])
         ctx.replace_all_inputs(node.output[0], last_node.output[0])  # ops=ctx.get_nodes()
+
+
+@tf_op("InvertPermutation")
+class InvertPermutationOp:
+    supported_dtypes = [
```

Comment:
This seems unused.  What is it for?

Reply:
The operator only supports these types. I was that on other operators. I assumed some verification would automatically happen. Based on your remark, I assume not. I'll do that then.

Reply:
Just searched the repo for `supported_dtypes`.  It is usually declared in a function and then iterated through for casting purposes.  I think you might have seen it and mistakenly thought it should be assigned in a class.  You defined it in RFFT, ComplexAbs, and Atan2 but I don't think it should be included.  Sorry I didn't catch that sooner.